linter/rules/field_name_is_not_snake_case.py

```diff
@@ -3,7 +3,7 @@
 from linter.helpers import is_snake_case
 
 
-class FieldNameIsSnakeCase(Rule):
+class FieldNameIsNotSnakeCase(Rule):
```

Comment:
I think this is okay to be FieldNameIsSnakeCase, right? Since the rule is that we _want_ the field to be snake-cased.